README.rst

```diff
@@ -551,6 +551,33 @@ For more information about OpenTok live streaming broadcasts, see the
 `Broadcast developer guide <https://tokbox.com/developer/guides/broadcast/>`_.
 
 
+Streaming audio to a websocket
```

Comment:
Please change "websocket" to "WebSocket" in the docs throughout. (But do not change the names in the API, such as `stream_audio_to_websocket`.)

Reply:
Have done this now. Note I changed the method name `stream_audio_to_websocket` to `connect_audio_to_websocket` as that fits the new product name better.